services/web/server/src/simcore_service_webserver/projects/projects_api.py

```diff
@@ -217,6 +221,29 @@ def get_delete_project_task(
 #
 
 
+async def _fetch_default_pricing_and_hardware_info(
```

Comment:
We generally try to use more standardised keywords like: get and set
```suggestion
async def _get_default_pricing_and_hardware_info(
```